pages/listGuests.vue

```diff
@@ -37,7 +37,14 @@
             </v-flex>
           </v-layout>
         </v-card-title>
+        <div v-if="loadingItems" class="mx-auto my-5" style="width: fit-content;">
```

Comment:
Looks good. What is the `width: fit-content` for?

Reply:
so that I can center the loading spinner using margin: auto, otherwise the width of the div will span the entire container and will not center the loading spinner